packages/core/scripts/hardhat/deployContracts.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+# The type of oracle system to set up, for example "beacon-l1" or "beacon-l2". Default is "dvm".
+ORACLE_TYPE=$1
```

Comment:
Could we just make higher-level tags (in addition to the existing ones), like beacon-l1, beacon-l2, dvm to avoid having to write this script? Or is that a misuse of tags?

Reply:
Yeah I just created `production`, `sink-oracle` and `source-oracle` tags for now. Will remove this script

Reply:
Awesome! Very cool. What does the production tag run? Both oracle deployments or just the source?

Reply:
production runs everything, but I'm thinking now we should change the name to "dvm" and not deploy either sink or source. "dvm" would be Finder, Registry, IdWhitelist, etc.